### PR TITLE
feat: run codecov in CI

### PR DESCRIPTION
**Pull request by hacdias**
License: MIT
Signed-off-by: Henrique Dias <hacdias@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ![](https://img.shields.io/badge/go-%3E%3D1.13.0-blue.svg)
 [![](https://travis-ci.com/ipfs/testground.svg?branch=master)](https://travis-ci.com/ipfs/testground)
+![Codecov](https://img.shields.io/codecov/c/gh/ipfs/testground)
```
Comment (daviddias) [inline README.md]:
```suggestion
[![Codecov](https://img.shields.io/codecov/c/gh/ipfs/testground)](https://codecov.io/gh/ipfs/testground)
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ![](https://img.shields.io/badge/go-%3E%3D1.13.0-blue.svg)
 [![](https://travis-ci.com/ipfs/testground.svg?branch=master)](https://travis-ci.com/ipfs/testground)
+[![Codecov](https://img.shields.io/codecov/c/gh/ipfs/testground)](https://codecov.io/gh/ipfs/testground)
```
Comment (daviddias) [inline README.md]:
```suggestion
[![codecov](https://codecov.io/gh/ipfs/testground/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/testground)
```

**Comment by daviddias** [comment]
image

17%!!

**Comment by nonsense** [comment]
I feel a bit uneasy about this PR - big up on the race detector, but I don't think we need `codecov` on Testground now.

Codecov would be introducing more friction than the value it adds at the current stage of the project.

**Comment by hacdias** [comment]
Hmm... Codecov probably adds < 0.1ms to the test time and adds info. We're not imposing any numbers for now

**Comment by hacdias** [comment]
We could simply remove the GitHub notification from the PR and keep the badge on the repo... Or tell Codecov not to "fail"

**Pull request merged**